### PR TITLE
[SPARK-514][SPARK-512] properties file test docker bug

### PR DESCRIPTION
**Pull request by ArtRand**
Changes `test_secrets` to use a file containing the spark properties to set. This file is passed using the `--properties-file` CLI flag. Also edited the Dockerfile so that bootstrap doesn't need to be downloaded (it's baked into the image).

Added docs for the properties file feature.

### CODE REVIEW

File: conf/spark-env.sh [hunk 1]
```diff
@@ -43,8 +42,6 @@ export LIBPROCESS_SSL_KEY_FILE=/mnt/mesos/sandbox/.ssl/scheduler.key
 export MESOS_MODULES="{\"libraries\": [{\"file\": \"libdcos_security.so\", \"modules\": [{\"name\": \"com_mesosphere_dcos_ClassicRPCAuthenticatee\"}]}]}"
 export MESOS_AUTHENTICATEE="com_mesosphere_dcos_ClassicRPCAuthenticatee"
 
-echo "spark-env: Environment" >&2
-env >&2
```
Comment (susanxhuynh) [inline conf/spark-env.sh]: Why did you remove this?

Comment (ArtRand) [reply conf/spark-env.sh]: I added that when I was debugging strict mode. I don't think a customer would necessarily want to spew the environment to the log every job.
